src/app/[lang]/onboarding/pick-avatar/avatarPicker.tsx

```diff
@@ -73,14 +86,14 @@ export default function AvatarPicker({
         accept="image/*"
         onChange={handleAvatarChange}
       />
-
       <div className="flex justify-center">
         <div className="relative h-[200px] w-[200px]">
           <button className="absolute right-0 top-0 z-10 rounded-full">
             <label htmlFor="your_avatar">
               <Image src="/plus-1.png" width={45} height={45} alt="plus" />
             </label>
           </button>
+
           <div className="h-[200px] w-[200px] overflow-hidden rounded-full">
             {customAvatar ? (
```

Comment:
The initial image can't be sent. 

Consider using something like that, so if an image isn't selected, use the first `defaultImages`
``` suggestion
    {customAvatar ? (
      <AvatarEditor
        className="-translate-x-7 -translate-y-7"
        ref={editorRef}
        image={customAvatar}
        width={205}
        height={205}
        scale={scale}
        borderRadius={100}
      />
    ) : (
      <Image
        src={`/${selectedAvatar}`}
        alt={`Selected Avatar ${selectedAvatar}`}
        width={200}
        height={200}
      />
    )}
```